README.md

```
@@ -1,146 +1,124 @@
 
-The *difference* between this...
+**`useFetch` could not refetch.** Its dependency array was `[]` while the body closed
+over `URI`, so the effect ran once and ignored every later URL. A custom fetch hook
+that cannot fetch a second URL is most of the hook missing. Create React App had been
+reporting it on every build:
 
-```javascript
-// Runs with every render
-useEffect(() => {
-})
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)